signac_dashboard/views.py

```diff
@@ -89,13 +103,34 @@ def get_file(dashboard, jobid, filename):
             abort(404, "The file requested does not exist.")
 
 
+def get_project_file(dashboard, filename):
+    if dashboard.project.isfile(filename):
+        mimetype = None
+        cache_timeout = 0
+        # Return logs as plaintext
+        textfile_regexes = ["job-.*\\.[oe][0-9]*", ".*\\.log", ".*\\.dat"]
```

Comment:
Remind me -- how are SLURM output files named by default? Does this first regex match that? I feel like this might have been a hack that I was using with a PBS scheduler... maybe it is no longer relevant? Are `.txt` files rendered in the browser or does it prompt to download those files? (This entire MIME type hack is probably something I made up just for myself...)

Reply:
I'm happy with removing the regex for `job-.*\\.[oe][0-9]*` if you think it's weird to have here. I think it was a hack to return PBS scheduler output/error files as plaintext but we don't have many PBS scheduler users anymore.